### DISCUSSION TITLE
Icon Request: icon-stop-sign

### DISCUSSION THREAD

**Discussion by anazar** [FortAwesome/Font-Awesome#21143] [Icon Requests]
A stop sign icon would be awesome!

**Comment by Luzifr** [comment]
A stop sign would definitely be nice! I really would need something like that, and i'm sure others would need it too! 

But i have one request: Please keep it international! A stop sign like this: 
![stop latin](https://f.cloud.github.com/assets/915591/1072507/d7e93dca-1492-11e3-9579-e469d2bb238e.jpeg)

… with a latin "STOP" in the middle is not internationally readable! Stick to this: 
![stop i18n](https://f.cloud.github.com/assets/915591/1072508/e83284ac-1492-11e3-9f8c-2e427817edf9.jpeg)

… and your sign is understandable around the world! ;-)

**Comment by tagliala** [comment]
I feel this one is a dup but I'm not able the find the original issue.

**Comment by poedenon** [comment]
I cant find a dup, would love a blank stop sign (octagon)

**Comment by madtown** [comment]
I would use this in warning messages so user would stop before proceeding and read text or would be good for inactive button states or something. Also would be great for stacking.

**Comment by camd** [comment]
yes please!  I could really use a stop-sign.  it's a little clearer than the tape recorder fa-stop when you want to indicate stopping a process of some kind.  Font Awesome is awesome...er... wonderful!  Even just a plain octagon would work.  But the hand looks cool, too.

**Comment by lenamtl** [comment]
+1

**Comment by diasalvatore** [comment]
+1

**Comment by Albert0Garcia** [comment]
+1
A stop sign and other traffic images such as a yield triangle & traffic light.

**Comment by jonkafton** [comment]
+1

**Comment by moonlimb** [comment]
\+ 1

**Comment by DUznanski** [comment]
+1, though if you go with international, it would probably be best if the hand's fingers are tightly together.

**Comment by ddeakes** [comment]
+1

**Comment by nshew13** [comment]
+1

I'm picturing a crosswalk's "do not walk" hand. Could those be separate symbols we layer using `fa-stack`?

**Comment by stlmikem** [comment]
+1

A series of traffic signs and lights, both American and European, would actually be a nice addition. Visually, they're recognizable in terms of symbolism (stop, caution, yield, etc.) as well as literal, depending on need.

**Comment by jklingen** [comment]
+1

**Comment by perrl01** [comment]
+1

**Comment by cmjchrisjones** [comment]
+1

**Comment by stimms** [comment]
+1

**Comment by dmland** [comment]
+1

**Comment by sfxian** [comment]
+1. Agreed the other stop related images have already been used by my system for other meanings. a stop sign would be the clearest icon for us.

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by kartik-joshi-01** [comment]
+1 we really need this (with duotone if possible)

**Comment by DjThossi** [comment]
+1
